rules/typescript.js

```diff
@@ -1,11 +1,14 @@
 module.exports = {
   rules: {
     '@typescript-eslint/array-type': ['error', { default: 'array-simple' }],
+    '@typescript-eslint/ban-ts-comment': 'error',
     '@typescript-eslint/consistent-type-definitions': ['error', 'interface'],
+    '@typescript-eslint/comma-spacing': 'off',
     '@typescript-eslint/explicit-function-return-type': [
       'error',
       { allowTypedFunctionExpressions: false },
     ],
+    '@typescript-eslint/explicit-module-boundary-types': 'error',
```

Comment:
This is a default, no need to add it explicitly - it's not done for other rules, so I'd keep it consistent: https://github.com/ridedott/eslint-config/blob/58780c910e7022e1b31c7f11adae911d81f4d9aa/index.js#L4

Reply:
fixed with https://github.com/ridedott/eslint-config/pull/314/commits/8ad156eb1bfc14ba3f60a7cd101ef0207f4422f6